generate.py

```diff
@@ -1405,7 +1390,7 @@ def copy_articles_and_assets(db, in_directory, out_directory):
         # Copy all images of all articles of the issue and downloads
         articles = [article for article in db.articles if article.issue_key == issue_key]
         article_index = 0
-        for article in articles:
+        for article in sorted(articles, key=lambda x: x.sort_index):
```

Comment:
nice would be a method on `db` that gets you articles
* filtered by x
* sorted by y
and use it throughout